builtins/xrootd/CMakeLists.txt

```diff
@@ -13,11 +13,16 @@ set(XROOTD_VERSION "5.5.2")
 set(XROOTD_PREFIX ${CMAKE_BINARY_DIR})
 message(STATUS "Downloading and building XROOTD version ${XROOTD_VERSION}")
 
+list(REMOVE_ITEM XROOTD_CLIENT_LIBRARIES OpenSSL::SSL)
```

Comment:
Is `ssl` not needed or is it the wrong spelling?

Reply:
It is needed, but only after `ExternalProject_Add`